eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -204,7 +204,7 @@ public SyncManager(GatewayContext context, ManagedTagProvider provider, String p
      * @param settings settings to use
      */
     public void startup(EwonConnectorSettings settings) {
-        logger.info("Starting Ewon sync manager.");
+        logger.info("Starting up...");
```

Comment:
This is a global comment for this PR.

When using ... to indicate a task is starting in the logs there must be a corresponding "done" log message to go along with it.

Reply:
... indicates that something is currently happening and you are waiting for it to finish.